solutions/ComplexNumberKoans.jl

```diff
@@ -0,0 +1,54 @@
+module ComplexNumberKoans
+
+#=
```

Comment:
You could move the links to the documentation over to the non-solution koans module, so that beginners starting to solve the exercises know exactly where to learn without having to look at the answers.